fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/converter/ViolationEntityToDtoConverter.java

```diff
@@ -43,6 +43,15 @@ public Violation convert(ViolationEntity source) {
 
         violation.setUsername(source.getUsername());
 
+        if (source.getApplication() != null) {
+            violation.setApplicationId(source.getApplication().getName());
```

Comment:
@harti2006 you are right, but the property will be set in the dto with this pull request ;) so we have not this property in the API right now.